.github/actions/create-jira-issues-for-community-activities/action.yml

```diff
@@ -33,10 +33,10 @@ runs:
       shell: pwsh
       run: |
         $output = "${{ github.event.pull_request }}" -eq "" ? "False" : (Confirm-PullRequestTitle "$($Env:PR_TITLE)")
-        Set-Output "is-issue-pr" $output
+        Set-Output -Key "is-issue-pr" -Value $output
```

Comment:
Are you sure positional parameters are not working? Same in .github/actions/publish-nuget/action.yml. Now we have three kinds of syntaxes being used for `Set-Output` throughout the repo.